src/audio/AudioResource.ts

```diff
@@ -2,6 +2,7 @@ import { Edge, findPipeline, StreamType, TransformerType } from './TransformerGr
 import { pipeline, Readable } from 'stream';
 import { noop } from '../util/util';
 import { VolumeTransformer } from 'prism-media';
+import { AudioPlayer } from './AudioPlayer';
```

Comment:
You might want to use `import type` to prevent circulars from happening, since AudioPlayer and AudioResource depend of each other now.
```suggestion
import type { AudioPlayer } from './AudioPlayer';
```